.husky/pre-commit

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx ts-node --esm mustache.ts
+# npx ts-node --esm mustache.ts
```

Comment:
isko comment kyu karna pada?

Reply:
This is not working and giving error. Last time mridul too had faced this issue. Can we connect to resolve this?